test/jdk/javax/swing/JTable/JTableScrollPrintTest.java

```diff
@@ -42,7 +42,7 @@
 /*
  * @test
  * @key headful
- * @bug 8210807
+ * @bug 8210807 8322140
```

Comment:
Suggestion:

You may update the test with `PassFailJFrame.builder()`.

I guess `Graphics2D g2d` object needs to be disposed.

Reply:
I guess here we don't have to dispose.

Reply:
Updated test with PassFailJFrame.builder().